src/test/regress/bin/normalize.sed

```diff
@@ -193,3 +193,6 @@ s/relation with OID [0-9]+ does not exist/relation with OID XXXX does not exist/
 
 # normalize storage id of columnar tables
 s/^storage id: [0-9]+$/storage id: xxxxx/g
+
+# ignore PL/pgSQL line numbers that differ on Mac builds
+s/(CONTEXT:  PL\/pgSQL function inline_code_block line )(\d+)/\1XX/g
```

Comment:
```suggestion
s/(CONTEXT:  PL\/pgSQL function inline_code_block line )([0-9]+)/\1XX/g
```

This works on my machine

Reply:
I made the change you suggested, and ran `./ci/normalize_expected.sh` and update all affected test outputs. Some tests still fail

Reply:
should we use `make reindent` instead ? (not sure if it helps)